setup.py

```diff
@@ -5,36 +5,45 @@
 
 # get the version (don't import mne here, so dependencies are not needed)
 version = None
-with open(op.join('pyriemann', '_version.py'), 'r') as fid:
+with open(op.join("pyriemann", "_version.py"), "r") as fid:
     for line in (line.strip() for line in fid):
-        if line.startswith('__version__'):
-            version = line.split('=')[1].strip().strip('\'')
+        if line.startswith("__version__"):
+            version = line.split("=")[1].strip().strip("'")
             break
 if version is None:
-    raise RuntimeError('Could not determine version')
+    raise RuntimeError("Could not determine version")
 
-with open('README.md', 'r', encoding="utf8") as fid:
+with open("README.md", "r", encoding="utf8") as fid:
     long_description = fid.read()
 
-setup(name='pyriemann',
-      version=version,
-      description='Riemannian Geometry for python',
-      url='https://pyriemann.readthedocs.io',
-      author='Alexandre Barachant',
-      author_email='alexandre.barachant@gmail.com',
-      license='BSD (3-clause)',
-      packages=find_packages(),
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      project_urls={
-          'Documentation': 'https://pyriemann.readthedocs.io',
-          'Source': 'https://github.com/pyRiemann/pyRiemann',
-          'Tracker': 'https://github.com/pyRiemann/pyRiemann/issues/',
-      },
-      platforms='any',
-      python_requires=">=3.6",
-      install_requires=['numpy', 'scipy', 'scikit-learn',  'joblib', 'pandas'],
-      extras_require={'docs': ['sphinx-gallery', 'sphinx-bootstrap_theme', 'numpydoc', 'mne', 'seaborn'],
-                      'tests': ['pytest', 'seaborn', 'flake8']},
-      zip_safe=False,
+setup(
+    name="pyriemann",
+    version=version,
+    description="Riemannian Geometry for python",
+    url="https://pyriemann.readthedocs.io",
+    author="Alexandre Barachant",
+    author_email="alexandre.barachant@gmail.com",
+    license="BSD (3-clause)",
+    packages=find_packages(),
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    project_urls={
+        "Documentation": "https://pyriemann.readthedocs.io",
+        "Source": "https://github.com/pyRiemann/pyRiemann",
+        "Tracker": "https://github.com/pyRiemann/pyRiemann/issues/",
+    },
+    platforms="any",
+    python_requires=">=3.7",
+    install_requires=["numpy", "scipy", "scikit-learn", "joblib", "pandas"],
+    extras_require={
+        "docs": [
+            "sphinx-gallery",
+            "sphinx-bootstrap_theme",
+            "numpydoc",
+            "mne",
+            "seaborn",
+        ],
+        "tests": ["pytest", "seaborn", "flake8"],
+    },
+    zip_safe=False,
```

Comment:
@sylvchev if we switch to black formatting we should enforce it in the CI.
Autoformatting creates big diffs that take time to reviewers for no good reason.

Reply:
Yes, completely agree. For this specific file, it was set up before CI and did not pass the linting test.
I could revert and use autopep8 or equivalent if you prefer.

Reply:
could you make sure that now this file is part of the linting?